extensions/wikia/UserLogin/EmailConfirmationController.class.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+class EmailConfirmationController extends WikiaController {
+	/**
+	 * Attempt to confirm the user's email address and show success or failure
+	 */
+	public function postEmailConfirmation() {
```

Comment:
Came back to review but I don't see a check for `wasPosted` here.  Has it been committed yet?